src/core/js/views/menuView.js

```diff
@@ -28,26 +29,32 @@ define([
         },
 
         isReady: function() {
-            if (this.model.get('_isReady')) {
-                _.defer(function() {
-                    $('.loading').hide();
-                    $(window).scrollTop(0);
-                    Adapt.trigger('menuView:ready', this);
-                    var styleOptions = { opacity: 1 };
-                    if (this.disableAnimation) {
-                        this.$el.css(styleOptions);
-                        $.inview();
-                    } else {
-                        this.$el.velocity(styleOptions, {
-                            duration: 'fast',
-                            complete: function() {
-                                $.inview();
-                            }
-                        });
-                    }
-                    $(window).scroll();
-                }.bind(this));
-            }
+            if (!this.model.get('_isReady')) return;
+
+            var performIsReady = function() {
+                $('.loading').hide();
+                $(window).scrollTop(0);
+                Adapt.trigger('menuView:ready', this);
+                $.inview.unlock('menuView');
+                var styleOptions = { opacity: 1 };
+                if (this.disableAnimation) {
+                    this.$el.css(styleOptions);
+                    $.inview();
```

Comment:
```suggestion
                    $.inview();
                    $(window).scroll();
                    return;
```
and drop the `else`?